clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/config/AuditLoggingConfiguration.java

```diff
@@ -46,11 +46,8 @@ public class AuditLoggingConfiguration {
   @JsonProperty("security_context")
   private SecurityContext securityContext;
 
-  @JsonProperty(value = "jvs_endpoint")
-  private String jvsEndpoint = "localhost:8080";
-
-  @JsonProperty(value = "require_justification")
-  private boolean justificationRequired = false;
+  @JsonProperty("justification")
+  private Justification justification = new Justification();
```

Comment:
Not quite familiar with Java. Do we need new Justification() here? Can we just have nil value as the default value?

Reply:
This will create a default justification object to keep the default values as they were. It is indifferent in terms of outcome comparing to set it to null.